### ISSUE TITLE
Email Users regarding deletion of files that don't play.

### ISSUE THREAD

**Issue by AndersEverett** [popuparchive/pop-up-archive#1327]
User emails listed here:
https://docs.google.com/spreadsheets/d/1mofzWZEkg9eayyjEvTerWkL5_-oxGsTkF-rJAXTuXrI/edit?ts=565c8d6f#gid=1232807286&vpid=A1

**Comment by annewootton** [comment]
Great. Assigning to @Saltzshaker to email everyone in this spreadsheet with the exception of rows highlighted green.

@AndersEverett: were there any items added to this doc from #1229? If so, I will take a closer look in case we want to save any of those. Otherwise, good to go.

**Comment by AndersEverett** [comment]
@annewootton  No, doc is the same as original. They were already included in that set.

**Comment by annewootton** [comment]
Great!
ᐧ

**Comment by Saltzshaker** [comment]
We are waiting on emailing and file deletion until the resolution of https://github.com/popuparchive/pop-up-archive/issues/1337. Moving to Megan.
